devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -256,8 +257,7 @@ private Map getBatteryResult(rawValue) {
 				def pct = batteryMap[volts]
 				if (pct != null) {
 					result.value = pct
-                    def value = pct
```

Comment:
not sure what the resolution here was. i can still see no `value` defined in this method. wouldn't that result in the description text always being null?